features/step_definitions/monitoring.rb

```diff
@@ -0,0 +1,79 @@
+Then(/^I check replicas of monitoring component for sno cluster$/) do
+
+  if infrastructure("cluster").infra_topology=="SingleReplica"
+
+    project("openshift-monitoring") 
+    #check deamonset replicas
+    step %Q/I run the :get client command with:/, table(%{
```

Comment:
@lihongyan1 I see a lot of repetitions which steps like that, which would indicate a good candidate for a generic step to replace them.

Reply:
@pruan-rht thx, but not sure what you mean

Reply:
@lihongyan1 with the step in PR #2042, you can replace lines 7-19 with something like
`Given there are 1 "daemon_set" replicas in the "openshift-monitoring" project`...same goes for the rest of the replica check.